### ISSUE TITLE
collect info: why did we fork twice today ?

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#100]
This task is about collecting infos, so it gets added into the bugtracker [here](https://github.com/gridcoin/Gridcoin-Research/issues)!

1. this morning we forked (A + B chain): "winner" was: A chain: [info](https://cryptocointalk.com/topic/1331-new-coin-launch-announcement-grc-gridcoin/page-1240#entry236591)
2. about 2h ago: the A chain from this morning "lost" against the new A2 chain: [info](https://cryptocointalk.com/topic/1331-new-coin-launch-announcement-grc-gridcoin/page-1241#entry236630)

In both cases what helped to get on the chain with the higher net weight was:
* just [restart the client](https://steemit.com/gridcoin/@erkan/out-of-sync-in-gridcoin-3-5-8-8)

questions to be answered:
* is this "only" a bug in the client and/or are we attacked?
* no attack? what kind of bug is it?

Keep in mind, before we had the problems starting ([March 20](https://cryptocointalk.com/topic/1331-new-coin-launch-announcement-grc-gridcoin/page-1236#entry236178)) our net weight levels were like 70-90 million in average, when many were staking like120-150 mio. 
* A + A2 chain from today had about each 30 million ranges
  * edit: A2 has now 70 million net weight (2017 April 5 7:50 AM, German time)
* see also: https://github.com/gridcoin/Gridcoin-Research/issues/106

Gridcoin version 3.5.8.8

**Comment by Mercosity** [comment]
Erkan, I didn't restart either of my win full nodes and they are on Block 861993 - Hash: 583b4 ... 9b90.
Mag = 0 on both nodes.

The question is which chain am I on? I must admit to be totally confused.

**Comment by Erkan-Yilmaz** [comment]
You are on the **A2 chain** when you have following hashes:

getblockhash 861993
hash = 583b4bdb8df62d7530d1b61cb4475fa1255c83fdf7ce9e66eae79d470fd09b90

getblockhash 861971
hash = 4e09582f9fd36ced05c06065ea16981eb172e803eb135e0e47c0217c242674b3

**Comment by Mercosity** [comment]
Ok, checked both hashes and they tally with the hashes for what you are calling the A2 chain.

I suppose I'll stick here for now and support this chain as I haven't restarted either node.

**Comment by TG13088** [comment]
Did see two anomalies in my log which may shed some clues  They are as follows:
**Anomaly #1**
04/04/17 18:13:52 

Signing Block for cpid 7ace61f1014802fe81b54a3010a9df33 and blockhash 96894542a4e640e9663239e3d8202ff577b0aa9d7aba48f545b4a8fcb11eb62d with sig MEUCIQDLDGPkt2+JAXb6di6J82sqkHHtAZMyPccX8H/vqsXHVwIgTlqQSJK9TgJHRh3zowtyFq62h1MSlIu0tE853boyLvo=

04/04/17 18:13:52 * Received block 867763db7df0dbf366e3571366160263e8ea4193951e8f1a2611c29555ffceb7; -

Signing Block for cpid 7ace61f1014802fe81b54a3010a9df33 and blockhash 96894542a4e640e9663239e3d8202ff577b0aa9d7aba48f545b4a8fcb11eb62d with sig MEQCIGIgFapnOH62f2UVIbcV/le7HX+kq+MgXmDHJWh3WJupAiBiHrSH3foXshQ/8Kzjtt2CyzfNYi++53m9po4isOX+bQ==

04/04/17 18:13:53 ERROR: ConnectBlock[] : Superblock hash does not match consensus hash; SuperblockHash: aa3920a997aa31f9dd0175711b7f241b, Consensus Hash: 
04/04/17 18:13:53 InvalidChainFound: invalid block=867763db7df0dbf366e3  height=862012  trust=3275838333972824660711873320762369734724920483558585417587751960956132392308  blocktrust=24484002312  date=04/04/17 18:13:36
04/04/17 18:13:53 InvalidChainFound:  current best=96894542a4e640e96632  height=862011  trust=3275838333972824660711873320762369734724920483558585417587751960931648389996  blocktrust=27946381969  date=04/04/17 18:11:12
04/04/17 18:13:53 ERROR: SetBestChain() : SetBestChainInner failed
04/04/17 18:13:53 ERROR: AcceptBlock() : AddToBlockIndex failed
04/04/17 18:13:53 ERROR: ProcessBlock() : AcceptBlock FAILED

.. to be continued

**Comment by TG13088** [comment]
**Anomaly #2**04/04/17 18:15:06  Received block b31668ccf4adead6474f87305b85788ecc4fd109b060f90b3dd489fc81dc3c22; -

**Reorganize**Postponing 0 reconnects
04/04/17 18:15:06 

Reorganizing Attempt #2.000000, regression to block #862012.000000 

04/04/17 18:15:06 Postponing 1 reconnects
04/04/17 18:15:06 

Reorganizing Attempt #3.000000, regression to block #862012.000000 

04/04/17 18:15:06 Postponing 1 reconnects
04/04/17 18:15:06 

Reorganizing Attempt #4.000000, regression to block #862012.000000 

04/04/17 18:15:06 Postponing 1 reconnects
04/04/17 18:15:06 

Reorganizing Attempt #5.000000, regression to block #862012.000000 

04/04/17 18:15:06 Postponing 1 reconnects
04/04/17 18:15:06 REORGANIZE
04/04/17 18:15:06 REORGANIZE: Disconnect 0 blocks; 96894542a4e640e96632..96894542a4e640e96632
04/04/17 18:15:06 REORGANIZE: Connect 1 blocks; 96894542a4e640e96632..867763db7df0dbf366e3
04/04/17 18:15:06 REORGANIZE Disc Size 0.000000REORGANIZE: done

**Comment by xXUnRealXx** [comment]
As to: 
getblockhash 861993
 hash = 583b4bdb8df62d7530d1b61cb4475fa1255c83fdf7ce9e66eae79d470fd09b90
I'm GOOD
getblockhash 861971
 hash = 4e09582f9fd36ced05c06065ea16981eb172e803eb135e0e47c0217c242674b3
I'm GOOD
Thus, I assume I'm on Chain A2

I've moved my wallet over to one of my slow pc's as the wallet tends to eat up 1 of 4 cores when loading and than staking. I've found it works better on the slow pc and it doesn't eat up as much CPU.  
Therefore I'll be leaving it staking at all times instead of me only running the wallet once every week or so. Hope that helps with the Network Stability.
